packages/integrations/mdx/package.json

```diff
@@ -53,6 +53,9 @@
     "unist-util-visit": "^4.1.2",
     "vfile": "^5.3.7"
   },
+  "peerDependencies": {
+    "astro": "workspace:^2.9.6"
```

Comment:
Should this become `3.0.0` before the release?

Reply:
I think changesets or pnpm will bump that before the release. We should definitely check our existing integrations also have that bumped before the initial release too.